docs/content/developers/building-contributing.md

```diff
@@ -48,6 +50,32 @@ sudo rm /usr/local/bin/ddev
 brew link --force ddev
 ```
 
+### Installing a downloaded binary in the $PATH
```

Comment:
We’ve almost got consistent title casing for headings on this page, so might as well continue here. And wrap `$PATH` in backticks since it’s a literal reference to a system variable. (Even though backtick styling in headings is kind of ugly.)

```suggestion
### Installing a Downloaded Binary in the `$PATH`
```